include/cutlass/epilogue/threadblock/fusion/visitor_load.hpp

```diff
@@ -524,12 +543,27 @@ struct VisitorColBroadcast {
     CUTLASS_DEVICE void
     begin_epilogue() {
       clear(tC_rCol);
+
       Tensor pred = make_tensor<bool>(shape(tC_gCol));
       CUTLASS_PRAGMA_UNROLL
       for (int i = 0; i < size(pred); ++i) {
         pred(i) = get<0>(tC_cCol(i)) < m;
       }
-      copy_if(pred, tC_gCol, tC_rCol);
+
+      if (params_ptr->ptr_col) {
+        // In this case we are loading from a column vector and broadcasting
```

Comment:
A design question: this isn't really a scalar operation anymore. Does it make sense to extend this visitor, or to add replace this with a vector broadcast instead that then has a broadcasting layout for its data

Reply:
>replace this with a vector broadcast instead that then has a broadcasting layout for its data

Could you point me to an example of this?

Reply:
thanks for taking a look at the PR BTW :)